### PR TITLE
Jennyf/claims

### PR DESCRIPTION
**Pull request by jennyf19**
remove .WithClaims() from application builder Issue #930

### CODE REVIEW

File: src/Microsoft.Identity.Client/AppConfig/IAppConfig.cs [hunk 1]
```diff
@@ -100,10 +100,6 @@ public interface IAppConfig
         /// </summary>
         IDictionary<string, string> ExtraQueryParameters { get; }
```
Comment (bgavrilMS) [inline src/Microsoft.Identity.Client/AppConfig/IAppConfig.cs]: What about ExtraQueryParameters  ?

**Comment by bgavrilMS** [comment]
@jennyf19 - before commiting, please rebase and reword the commit messages. The current ones are not descriptive enough (actually you probably want to merge the 2 commits into a single one, but still, a better commit message would be nice).

File: src/Microsoft.Identity.Client/AppConfig/IAppConfig.cs (hunk 1 above)
Comment (jennyf19) [reply src/Microsoft.Identity.Client/AppConfig/IAppConfig.cs]: @bgavrilMS what about them?

Comment (bgavrilMS) [reply src/Microsoft.Identity.Client/AppConfig/IAppConfig.cs]: I was just wondering if they make sense at this level. I think they do...

Comment (jennyf19) [reply src/Microsoft.Identity.Client/AppConfig/IAppConfig.cs]: @bgavrilMS Oh...I see. you mean should we leave them here or move them like claims? I agree with you, I think they make sense to leave here on the application object. @jmprieur thoughts?

Comment (jmprieur) [reply src/Microsoft.Identity.Client/AppConfig/IAppConfig.cs]: They do make sense both at the app level and the acquire token level
for instance at the app level: slices
at the acquiretoken level: mkt=fr-fr

**Pull request merged**